### PR TITLE
progress: silence warning

### PR DESCRIPTION
**Pull request by gcongiu**
## Author Checklist
* [ ] Reference appropriate issues (with "Fixes" or "See" as appropriate)
* [x] Commits are self-contained and do not do two things at once
* [x] Commit message is of the form: `module: short description` and follows [good practice](https://chris.beams.io/posts/git-commit/)
* [ ] Add comments such that someone without knowledge of the code could understand
* [x] Have read and agree to the Yaksa CLA terms (https://github.com/pmodels/yaksa/wiki/Yaksa-Contributor-License-Agreement)

### CODE REVIEW

**Comment by gcongiu** [comment]
test:yaksa

File: src/backend/src/yaksuri_progress.c [hunk 1]
```diff
@@ -139,7 +139,7 @@ static int alloc_subop(progress_subop_s ** subop)
     yaksuri_request_s *request_backend = (yaksuri_request_s *) elem->request->backend.priv;
     yaksuri_gpudev_id_e id = request_backend->gpudev_id;
     bool need_device_tmpbuf = false, need_host_tmpbuf = false;
-    int devid;
+    int devid = 0;
```
Comment (pavanbalaji) [inline src/backend/src/yaksuri_progress.c:142]: Can you initialize this to `INT_MIN` instead, so the code will bomb out if we accidentally use it?

**Comment by gcongiu** [comment]
test:yaksa

**Comment by gcongiu** [comment]
test:yaksa

**Comment by pavanbalaji** [comment]
Can you change the commit message which still says `0`?

**Comment by gcongiu** [comment]
test:yaksa

**Pull request merged**